Formula/terragrunt.rb

```diff
@@ -1,9 +1,9 @@
 class Terragrunt < Formula
   desc "Thin wrapper for Terraform e.g. for locking state"
   homepage "https://github.com/gruntwork-io/terragrunt"
-  url "https://github.com/gruntwork-io/terragrunt/archive/v0.18.3.tar.gz"
-  sha256 "b38c7f72ece72bfbe719486f4196c225a02d5e0c1345ba8ec6dacadb8eb58c59"
-  head "https://github.com/gruntwork-io/terragrunt.git"
+  url "https://github.com/gruntwork-io/terragrunt.git",
+      :tag      => "v0.18.5",
+      :revision => "b432bcf95885b546a7f59cf58a32da0f39de0e83"
```

Comment:
We prefer to build from source tarball, can you switch back to that?

Reply:
Sure, I will switch back to use tarball.